tests/components/google_generative_ai_conversation/test_conversation.py

```diff
@@ -64,46 +68,70 @@ async def test_default_prompt(
         mock_config_entry,
         options={**mock_config_entry.options, **config_entry_options},
     )
+    entities = []
+
+    def create_entity(device: dr.DeviceEntry) -> None:
+        """Create an entity for a device and track entity_id."""
+        entity = entity_registry.async_get_or_create(
+            "light",
+            "test",
+            device.id,
+            device_id=device.id,
+            original_name=str(device.name),
```

Comment:
We guard for devices with integers as names. That's invalid but done by custom component. We had this break OpenAI in the past: https://github.com/home-assistant/core/pull/88047